docs/install/uninstall.md

```diff
@@ -6,40 +6,40 @@ To uninstall your Coder server, delete the following directories.
 
 ## Cached Coder releases
 
-```console
+```shell
 rm -rf ~/.cache/coder
 ```
 
 ## The Coder server binary and CLI
 
 Debian, Ubuntu:
```

Comment:
Should we use tabs for these?

Reply:
I tried tabs but it looks better without them